common/src/main/java/org/apache/carbondata/common/logging/impl/ExtendedRollingFileAppender.java

```diff
@@ -203,19 +194,16 @@ public void rollOver() {
   private void cleanUpLogs(final String startName, final String folderPath) {
     if (maxBackupIndex > 0) {
       // Clean the logs files
-      Runnable r = new Runnable() {
-
-        public void run() {
-          synchronized (ExtendedRollingFileAppender.class) {
-            cleanupInProgress = true;
-            try {
-              cleanLogs(startName, folderPath, maxBackupIndex);
-            } catch (Throwable e) {
-              // ignore any error
-              LogLog.error("Cleaning logs failed", e);
-            } finally {
-              cleanupInProgress = false;
-            }
+      Runnable r = () -> {
```

Comment:
original one maybe more friendly to read

Reply:
Reverted